newspaper-api.yml

```diff
@@ -22,20 +22,25 @@ metadata:
 spec:
   ports:
     # the port that this service should serve on
-  - port: 38765
+  - port: 80
```

Comment:
did you check that's 80?  where can I check that?

Reply:
yes sure... change your kubectl context to `smarp-cluster` and run `kubectl describe service newspaper-api`